SimpleBrowser/Browser.cs

```diff
@@ -289,11 +289,19 @@ public void ClearException()
 			LastWebException = null;
 		}
 
-		public void ClearWindows()
+		public void ClearWindowsInContext()
 		{
             foreach (var window in _allWindows.ToArray()) window.Close();
 			_allWindows.Clear();
 		}
+        public static void ClearWindows()
+        {
+            foreach (var list in _allContexts.ToArray())

```

Comment:
You need to add the removal of the local copies
Maybe ClearWindowsInContext() or similar